src/routes/plans/+page.svelte

```diff
@@ -245,6 +245,10 @@
     }
   });
 
+  onDestroy(() => {
+    resetPlanStores();
```

Comment:
Why reset the plan stores here?

Reply:
It was to clear the plan error, much like we did for models. At the time I tested it, there was a backend bug that prevented the plan from being created when the model contained goals with priorities that were not added in a certain order. That error showed up on the plan creation page. I navigated to the model to remove the goals, and came back to the plan creation page and still saw the error persisted.